pkg/oc/lib/describe/describer_test.go

```diff
@@ -87,10 +82,13 @@ var MissingDescriberCoverageExceptions = []reflect.Type{
 }
 
 func TestDescriberCoverage(t *testing.T) {
+	scheme := runtime.NewScheme()
+	kubernetesscheme.AddToScheme(scheme)
+	api.Install(scheme)
```

Comment:
nit: util.Must()

Reply:
These are tests so it'll fail one way or the other, I don't care that much.